controllers/storagecluster/storagecluster_controller.go

```diff
@@ -85,6 +86,9 @@ type StorageClusterReconciler struct {
 	recorder           *util.EventReporter
 	OperatorCondition  conditions.Condition
 	IsNoobaaStandalone bool
+
+	// client required for making grpc call to the external OCS storage cluster
+	externalClusterClient *externalClient.OCSProviderClient
```

Comment:
As discussed, this should not be used anymore, can we remove it?

Reply:
done

Reply:
done